app/demo-geant-integration/HepMC3Reader.cc

```diff
@@ -67,11 +95,14 @@ void HepMC3Reader::GeneratePrimaryVertex(G4Event* g4_event)
     {
         const auto& part_data = gen_particle->data();
 
-        if (part_data.status != 1)
+        if (part_data.status <= 0)
         {
             // Skip particles that should not be tracked
             // Status codes (page 13):
             // http://hepmc.web.cern.ch/hepmc/releases/HepMC2_user_manual.pdf
+            CELER_LOG_LOCAL(info)
+                << "Skipped status code " << part_data.status << " for "
+                << part_data.momentum.e() << "MeV primary";
```

Comment:
We probably need a space here:
```suggestion
                << part_data.momentum.e() << " MeV primary";
```